.trellis/scripts/common/task_utils.py

```
@@ -0,0 +1,274 @@
+    # Check empty or null
+    if not normalized or normalized == "null":
+        print("Error: empty or null task path", file=sys.stderr)
+        return False
+
+    # Reject absolute paths
+    if Path(task_path).is_absolute():
+        print(f"Error: absolute path not allowed: {task_path}", file=sys.stderr)
+        return False
+
+    # Reject ".", "..", paths starting with "./" or "../", or containing ".."
+    if normalized in (".", "..") or normalized.startswith("./") or normalized.startswith("../") or ".." in normalized:
```

Ruff (0.15.18)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)